code/__DEFINES/preferences.dm

```diff
@@ -5,7 +5,7 @@
 #define SOUND_AMBIENCE (1<<2)
 #define SOUND_LOBBY (1<<3)
 #define MEMBER_PUBLIC (1<<4)
-#define INTENT_STYLE (1<<5)
+// #define INTENT_STYLE (1<<5) - intents no longer exist
```

Comment:
is there a reason to comment it rather than removing it

Reply:
no

Reply:
actually *sort of*, it's so that it doesnt overlap when we add a new preference here, but i'd rather we just do that tbqh

Reply:
yeah that was the intent (no pun intended) as a sort of "just in case the preference is reused in the future saved preferences might be slightly off at the start"

Reply:
i'll allow it